R/derive_params_growth_age.R

```diff
@@ -296,7 +300,7 @@ derive_params_growth_age <- function(dataset,
         mutate(
           AVAL := case_when( # nolint
             AVAL > 3 ~ 3 + ({{ analysis_var }} - SD3pos) / (SD3pos - SD2pos),
-            AVAL < -3 ~ -3 + ({{ analysis_var }} - SD3neg) / (SD2neg - SD3neg),
+            AVAL < -3 ~ -3 - ({{ analysis_var }} - SD3neg) / (SD2neg - SD3neg),
```

Comment:
![image](https://github.com/user-attachments/assets/0c7b3b41-91d7-42af-bd88-dd1e9f9549fc)

It was correct as is, I followed the formulation directly from the textbook, David uses 
`z < (-3), -3 - abs((var - sdm3)/(sdm2 - sdm3))`

which is the same as long as the right side fractional term is negative

Reply:
@zdz2101 I like your style more than what I used!